contrib/sacrebleu/sacrebleu.py

```diff
@@ -37,7 +37,7 @@
 import math
 import unicodedata
 
-VERSION = '1.3.0'
+VERSION = '1.2.10'
```

Comment:
not sure if sneaking this in is the right way. Why not make a small separate PR and explain the downgrade?